tidy3d/web/api/webapi.py

```diff
@@ -885,7 +885,7 @@ def real_cost(task_id: str, verbose=True) -> float:
     Examples
     --------
 
-    To obtain the cost of a simulation, you can use the function ``tidy3d.web.real_cost(task_id)``. In the example
+    To obtain the cost of a simulation, you can use the function ``tidy3d.web.api.webapi.real_cost(task_id)``. In the example
```

Comment:
probably the user would just call `tidy3d.web.real_cost(task_id)`. This function should be imported at `web/__init__.py`, so maybe best to display it this way in the docstring

Reply:
Ahh I see what's happened. I'll change this now, but I think maybe this has implications for the autosummary overall, but will deal with it post rc1.

Reply:
yea in the auto summary it seems to only work with the full path to the function `web.api.webapi.real_cost`) I was never sure why exactly. but yea low priority.

Reply:
although would be nice to eventually display it like we want them to call it